rust_icu_ucal/src/lib.rs

```diff
@@ -413,21 +410,12 @@ mod tests {
         Ok(())
     }
 
-    /// Returns a `UDateFormat` for ISO 8601 dates of the form "2020-05-01T23:01:34.842+17:00".
-    fn iso_8601_format() -> Result<UDateFormat, common::Error> {
-        // Locale and tz don't matter for this pattern.
-        let locale = ULoc::try_from("und")?;
-        let tz = UChar::try_from("UTC")?;
-        let pattern = UChar::try_from("yyyy-MM-dd'T'HH:mm:ss.SSSZZZZZ")?;
-        UDateFormat::new_with_pattern(&locale, &tz, &pattern)
-    }
-
     #[test]
     fn test_set_date() -> Result<(), common::Error> {
-        let date_format = iso_8601_format()?;
-
-        let time_a = date_format.parse("2020-05-07T21:00:00.000-04:00")?;
-        let time_b = date_format.parse("2020-05-04T21:00:00.000-04:00")?;
+        // 2020-05-07T21:00:00.000-04:00
```

Comment:
Consider also adding a note why this is not parsed.

Reply:
Done